### DISCUSSION TITLE
There are a few optimization suggestions on the UI

### DISCUSSION THREAD

**Discussion by ZKANGGIT** [apache/skywalking#12329] [General]
<img width="619" alt="企业微信截图_1718265899524" src="https://github.com/apache/skywalking/assets/36831988/c871000f-771b-4201-926a-69a1979742e4">

The node that reports the error is inconspicuous and difficult to find

**Comment by wu-sheng** [comment]
Tracked @ https://github.com/apache/skywalking/issues/12330
